### ISSUE TITLE
Write a Code of Conduct for the AutSPACEs platform 

### ISSUE THREAD

**Issue by GeorgiaHCA** [alan-turing-institute/AutisticaCitizenScience#566]
## Summary

This is an issue for the draft code of conduct following community consultation.

We will be writing the code of conduct here: https://docs.google.com/document/d/1qhBlUxMVv8F_QGuY1re2xbLLZ-mNmak4TISmC-nFW58/edit?usp=sharing

Previous draft here: https://docs.google.com/document/d/1_gyWhJ5Zdv8nq2A5GeuIWQzdtVsW0MiUs0EDBbaB-bo/edit?usp=sharing 

## Date Due by

Thursday 28th October 

## Details

Code of Conduct should include: 

* Scope
* WHAT THE PLATFORM IS (AND ISN'T) FOR
* e.g. cannot offer professional help, but can link to appropriate sources; not for parental venting etc. 
* SECTION ON HOW YOU RESPOND OTHERS' STORIES 
* Explanation of why there is no comments section: and rule that you cannot write a response to another story for this reason
* Discuss the ability to share publicly and with research: only public comments will be moderated
* Do not copy/scrape/or post 
* SECTION ON SUBMITTING YOUR OWN STORIES 
* Basic principles of what is and is not acceptable for submission: this willl red light
* List of things which are likely to be triggering/may not suitable for everyone to read: this will be orange light
* SECTION ON GUIDELINES FOR OTHERS
* Guidelines for entering on behalf of other people as a section 
* REPORTING BREACHES OF CODE OF CONDUCT AND SAFEGUARDING

## Suggestions for tasks that can fix this issue

Let's do a rough code of conduct with some ideas in each section, reay to share with the community and iterate. This is not a final version, but a first start!

**Comment by skfantoni** [comment]
Hey, where can I find the community feedback from draft one?

**Comment by skfantoni** [comment]
Having read through all of the CoC sections from the moderation groups, I've found some key points and also have some questions that I think we need to answer. 

- [ ] What _do_ we mean when using the term 'neurology'?
- [ ] How can we make caregivers feel welcome whilst making it clear where the barrier is?
- [ ] How do we clearly lay out the 'rules' to those who don't naturally adhere to them without isolating or 'judging'?

**Comment by skfantoni** [comment]
Questions based on current CoC:

- [ ] Are all moderators going to be autistic?
I think it's been agreed that Kirstie is at the top of the moderation chain, so it's it fair to make this promise?
- [ ] I assume A/T citizen science platform is now autSPACEs
- [ ] Is 'SPACEs' and acronym? If so could we include an explanation of the project name in the second section of the new CoC?
- [ ] Are we intending to add graphics to the CoC? 
I think my summary of the 3 purposes (processing by writing, understanding by reading, contributing to research) of the sight would work well as a graphic and help people understand what the project is for.

**Comment by skfantoni** [comment]
I have added an edited version of the original introduction to the new CoC. I've cut it down and simplified it whilst also making it slightly more direct (ie. will be expected to follow... changed to is required to follow...). Let m know what you think.

**Comment by skfantoni** [comment]
I've some jiggling with the order and names of section titles but left them as suggestions as I know the doc is still a guide for us.

**Comment by GeorgiaHCA** [comment]
Heya - thanks for your careful thinking and hard work on this Susanna! To respond to some of your queries:

> [ ]  What _do_ we mean when using the term 'neurology'?

This term comes from the [code of conduct](https://www.autangel.org.uk/wp-content/uploads/2021/09/2103-AutAngel-code-of-conduct.pdf) for AutAngels which is an autistic community. I adapted it following the advice of Caroline Hearst after a talk of hers I went to, to give a rough start. It is a little confusing as a term. I think it is around respecting the unique and diverse different brains people have, and avoidinig neurotypical bias. How would you want to define/rephrase it?

**Comment by GeorgiaHCA** [comment]
>  Are all moderators going to be autistic?
I think it's been agreed that Kirstie is at the top of the moderation chain, so is it fair to make this promise? 

I think that it is a lot of responsiblity to put on a volunteer to be ultimately responsible for safeguarding/moderation  decisions? Perhaps what we can commit to is having an autistic person involved in all moderation decisions?

**Comment by GeorgiaHCA** [comment]
> I assume A/T citizen science platform is now autSPACEs
> Is 'SPACEs' an acronym? If so could we include an explanation of the project name in the second section of the new CoC?

Yep - AutSPACEs with a capitol 'A'. "SPACEs" stands for researchng Sensory Processing for Accessible Community Environments

**Comment by GeorgiaHCA** [comment]
>Are we intending to add graphics to the CoC?
>I think my summary of the 3 purposes (processing by writing, understanding by reading, contributing to research) of the site would work well as a graphic and help people understand what the project is for.

I think this is a fab idea! Yes! Definitely the community feedback includes a lot of preferences for graphics, and it makes it more clear and accessible. I think another graph shownig the moderation steps might also be useful?

**Comment by GeorgiaHCA** [comment]
> I have added an edited version of the original introduction to the new CoC. I've cut it down and simplified it whilst also making it slightly more direct (ie. will be expected to follow... changed to is required to follow...). Let m know what you think.

I think this is great Susanna. I think it is warm in tone and clear

**Comment by GeorgiaHCA** [comment]
>How can we make caregivers feel welcome whilst making it clear where the barrier is?
>How do we clearly lay out the 'rules' to those who don't naturally adhere to them without isolating or 'judging'?

These are really excellent questions, which I think gets to the heart of why a bespoke CoC for AutSPACEs is so important. 

My view for the first point is that we can be very explicit about valuing the input of caregivers, and welcome them and their support, and we can also explain the reasons for any restrictions. We can also include links to other places where caregivers can go for support?

The second one is so tricky. I think it's really important that we don't encode any arbitrary, neurotypical norms, and that we encourage diverse forms of expression. Again, I think that being very explicit about this is important. We should also have very clear reasons why we will not accept, or we want to discourage, certain behaviours. If there are no clear reasons why a rule is there, we should remove it from the code of conduct, as it may just be reflecting neurotypical bias. 

A final point would be to welcome feedback from all, and be open to iterating the code of conduct. Recognising room for learning and that we do not have all the answers will change the power dynamic, and give people an opportunity to be represented. 

These are just my views! Would be curious what others think. Again, your deep thinking and reflection is hugely appreciated @skfantoni

**Comment by skfantoni** [comment]
> code of conduct

Had a late night motivation boost. I've started putting all the discriminatory topics into a table. I'm wondering if we could just use the term 'Neurodiversity'. It'll be included in our glossary and doesn't cut out neurotypical people just emphasises that there's not only one way of thinking.

**Comment by skfantoni** [comment]
> Heya - thanks for your careful thinking and hard work on this Susanna! To respond to some of your queries:
> 
> > [ ]  What _do_ we mean when using the term 'neurology'?
> 
> This term comes from the [code of conduct](https://www.autangel.org.uk/wp-content/uploads/2021/09/2103-AutAngel-code-of-conduct.pdf) for AutAngels which is an autistic community. I adapted it following the advice of Caroline Hearst after a talk of hers I went to, to give a rough start. It is a little confusing as a term. I think it is around respecting the unique and diverse different brains people have, and avoidinig neurotypical bias. How would you want to define/rephrase it?

Had a late night motivation boost. I've started putting all the discriminatory topics into a table. I'm wondering if we could just use the term 'Neurodiversity'. It'll be included in our glossary and doesn't cut out neurotypical people just emphasises that there's not only one way of thinking.

**Comment by skfantoni** [comment]
How do we intend to respond to stories that include comparisons? 

- Comparing your experiences to someone you know
- Comparing your experiences to another story

I feel this is something that may arise when people discuss their own sensory differences. If there's no identifying factors when describing the 'other person' is this ok? Or do we consider it writer on behalf of someone without consent?

**Comment by skfantoni** [comment]
I've been looking through the Focus Group Summaries to try and flesh out an idea of why users may be coming to our platform and I feel like I've hit a brick wall when it comes to those reporting on behalf of others. Very little of the information collated focuses on them, which is understandable, however, I think we need to know who will be using the platform and why in order to complete the CoC. 

Without further research, I would suggest that at this point we either go into it blind (leave the section out of the CoC and learn as we go) or make the platform first-hand reports only.

**Comment by skfantoni** [comment]
I've completed the new [Autistic Person personas ](https://docs.google.com/document/d/166ATKNiqsSnIUGsOWr2-muMMXNxKYN1So51jJ0DPOvg/edit?usp=sharing). [These](https://docs.google.com/document/d/1KC8qahyWL8OjeHoj7LlKkYbE_RCRxZQVGd8xxNtcLUU/edit?usp=sharing) are the notes I took from the focus groups summaries and based these personas off of. 

I think we need to really think about comparisons and where we draw the line in regards to 'responding' to posts. I will discuss this in greater detail at the meet-up on 23rd December.

**Comment by skfantoni** [comment]
Had a quick thought as I was getting ready for bed, where do we stand on brand names and directly mentioning products? Ie. brand of ear-plugs that helped with audio issues, or a specific pair of shoes that make someone more comfortable

**Comment by GeorgiaHCA** [comment]
Hmm that's a good point @skfantoni ! Is your concern that people would use the platform to promote stuff? I think many people could find specific recommendations useful, and we could also tweak the code of conduct if we suspected it was being misused in that way. Would appreciate your thoughts on this @KirstieJane and @JamesSCTJ?

**Comment by skfantoni** [comment]
> Is your concern that people would use the platform to promote stuff? I think many people could find specific recommendations.

My thoughts precisely. Again like with many things I just feel there is a very thin line between recommending and promoting. Affiliates of brands may use the site to push products. Individuals may simply mention them. You can't tell which is which, so knowing where we stand is important. 

Of course, long term, there may be chances of the project making deals with brands or brands approaching us (don't think this is in the plan but it could happen). It would bring in support but damage the integrity of the project.

**Comment by skfantoni** [comment]
### CoC Update 27.1.22

I'm currently working on the CoC. I've completed the first draft for the 'RESPONDING TO OTHERS' STORIES' section.

What needs to be done:

- [ ] Complete WRITING ON BEHALF OF OTHERS section
- [ ]  Write examples to distinguish between 'red' and 'amber'
- [ ] Create a list of 'Sensitive data'
- [ ] Decide where the information on 'difference between public and research sharing' should go.

**Comment by GeorgiaHCA** [comment]
I completed the guidelines on behalf of others section based on our discussions using questions as prompts and giving some examples. Tried to edit it down and make it reasonably clear. if you could take a look @JamesSCTJ and @skfantoni that would be amazing!

**Comment by skfantoni** [comment]
I have just added the box of sensitive data examples. I've added it directly below the first paragraph regarding what is red.

@GeorgiaHCA In regard to the 'Writing on behalf of others' section, I think it's great. Definitely couldn't have worded it that well myself. Is it ok if I jiggle the order around a bit? I just feel it would flow better if we switched the last two sections, or even put them at the top seeing as they are key. I may also edit the intro as you left my original version in and it's a bit repetitive and doesn't need both parts.

**Comment by GeorgiaHCA** [comment]
Thanks for the feedback @skfantoni. Glad you like the guidelines wording. Definitely feel free to change the order of information for better flow. I trust your judgement!

**Comment by GeorgiaHCA** [comment]
Complete! We can add new issues for iterating following user testing. Fabulous work everyone!
